spec/integration/navigation_spec.rb

```diff
@@ -1,8 +1,8 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.